test/config/config-logging.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-logging
+  namespace: knative-serving
+  labels:
+    serving.knative.dev/release: devel
+
+data:
+  zap-logger-config: |
```

Comment:
I think you can leave this out entirely to pick up the default? Not entirely sure though.

Reply:
I just tried applying this yaml without this block and the `zap-logger-config` was not set

Reply:
`NewConfigFromConfigMap` function will not pick up logging config without `zap-logger-config` key. @markusthoemmes were you talking about labels or the `zap-logger-config` key?

cc @andrew-su

Reply:
My thought was that we'd only need the specific loglevels below and that the broader loggingconfig would be defaulted in.

Reply:
If both are kubectl applied, then applying this without it will strip it in the three way merge.